configure.ac

```diff
@@ -648,6 +648,18 @@ AS_IF([test "x$enable_yaml" != "xno"], [
 ])
 AM_CONDITIONAL(HAVE_LIBYAML, test "x$have_libyaml" = xyes)
 
+AH_TEMPLATE([HAVE_ASPELL],
+	[Define this value if aspell is available.])
+AC_ARG_ENABLE([aspell],
+	[AS_HELP_STRING([--disable-aspell],
+		[enable aspell support])])
```

Comment:
... sigh. Thank you. Spell checking is not perfect for foolish mistakes. 

I will make a fixup commit.